Library/Styles/CheckoutStylesTests.swift

```diff
@@ -39,6 +39,15 @@ final class CheckoutStylesTests: XCTestCase {
     XCTAssertEqual(9, stackView.spacing)
   }
 
+  func testCheckoutSwitchControlStyle() {
```

Comment:
I really don't think these tests are necessary 😬 Anytime we decide to change a small aspect of the checkout UI these tests break - it seems like it adds unnecessary maintenance for something that's already covered with other tests (snapshot tests) and offers only a marginal increase in test coverage.

Reply:
**Deleted**